final-project/templates/product.html

```diff
@@ -0,0 +1,29 @@
+<!--Final Project | product | (jwq1)-->
+
+{% extends "layout.html" %}
+
+{% block title %}
+    {{product_name}}
+{% endblock %}
+
+{% block main %}
+    <header>
+        <h1>{{product_name}}</h1><h5>{{brand}}</h5>
+    </header>
+    <img src="{{image}}" alt="Picture of {{product_name}}" height=350>
+    <div>
+        <h4>{{price}}</h4>
+    </div>
+
+    <h1>Description</h1>
+    <p>{{description}}</p>
+
+    <h2>Characteristics</h2>
+    <div>
+        <ul class = unformat>
```

Comment:
Does this even work? There are no quotes.

Reply:
🚫 nope

Reply:
I updated this to be less confusing, and added a TODO

🏁 done